doc/whats-new.md

```diff
@@ -1,356 +1,197 @@
 
-As usual, lots of small bugs have been fixed. More than 60 pull requests were
-closed in oq-hazardlib and more than 160 pull requests were closed in
-oq-engine. For the complete list of changes, please 
-see the changelogs: https://github.com/gem/oq-hazardlib/blob/engine-2.2/debian/changelog and https://github.com/gem/oq-engine/blob/engine-2.2/debian/changelog.
+More than 20 pull requests were closed in oq-hazardlib and more than
+70 pull requests were closed in oq-engine. For the complete list of
+changes, please see the changelogs:
+https://github.com/gem/oq-hazardlib/blob/engine-2.3/debian/changelog
+and https://github.com/gem/oq-engine/blob/engine-2.3/debian/changelog.
 
 A summary follows.
 
-General improvements
-====================================
+New features in the engine
+--------------------------
+
+From this release it is possible to take into account the *topography
+of the region of interest*. It is enough to write the elevation of the
+hazard sites in the sites file or in the site model file, as a
+negative depth, and the engine will compute the ground shaking
+correctly. Before there was no way to specify the elevation of the hazard sites
+and the ground shaking was computed only at the level of the sea.
 
-Python 3 support
+From this release it is also possible to use *magnitude-dependent
+integration distances*. This is expected to have a substantial impact on the
+calculation time, since small magnitude ruptures will affects a lot
+less sites than before. In order to use the feature the user has to
+specify in the `job.ini` file a list of pairs (magnitude, distance) for each
+tectonic region type. In the past the user would write
+
+`maximum_distance = {'Active Shallow Crust': 200}`
+
+and the engine would use an integration distance of 200 km for all
+ruptures of the given tectonic region type, disregarding the magnitude.
+Nowadays, the user can write
+
+`maximum_distance = {'Active Shallow Crust': [(8, 200), (7, 100), (5, 20)]}`
+
+and the engine will use an integration distance of 200 km for ruptures
+of magnitude 8, of 100 km for ruptures of magnitude 7, and of 20 km
+for ruptures of magnitude 5. Intermediate magnitudes will use a
+linearly interpolated distance. It is up to the user to provide a
+sound magnitude-distance function. For the future, we plan to release
+tools to help the users in the task of inferring magnitude-distance functions.
+If in doubt, you can always use the old syntax, equivalent to a
+constant maximum distance for all magnitudes, and the engine will keep
+working as in the past.
+
+A lot of work went into the UCERF calculators, but they are still officially
+marked as experimental and are left undocumented on purpose. There is a
+new time-dependent classical calculator, while the old calculators were
+substantially improved.
+
+There is an experimental command `run_tiles` to split a `sites.csv`
+file in tiles and run multiple calculations at once. This for expert
```

Comment:
I suggest "This too is for use by experts only." but perhaps it would be useful to say how one can find out more or if we really don't want to talk about it, to not mention it in the what's new.  Also, should we rename run_tiles to experimental_run_tiles (or at least show a scary "This is experimental" message) ?